maestrowf/maestro.py

```diff
@@ -436,6 +445,10 @@ def setup_argparser():
     status.add_argument(
         "directory", type=str, nargs="+",
         help="Directory containing a launched study.")
+    status.add_argument(
+        "--layout", type=str, choices=['flat', 'narrow', 'legacy'],
```

Comment:
I believe this will error out if not in the choices and is automatically handled by the argument parser class.